$OpenBSD: patch-tools_clang_include_clang_Basic_DiagnosticSemaKinds_td,v 1.1 2017/02/12 11:25:19 ajacoutot Exp $

Disable -Wpointer-sign warnings per default

base gcc does the same.

--- tools/clang/include/clang/Basic/DiagnosticSemaKinds.td.orig	Sun Feb 12 04:37:29 2017
+++ tools/clang/include/clang/Basic/DiagnosticSemaKinds.td	Sun Feb 12 04:39:28 2017
@@ -6534,7 +6534,7 @@ def ext_typecheck_convert_incompatible_pointer_sign : 
   "sending to parameter of different type}0,1"
   "|%diff{casting $ to type $|casting between types}0,1}2"
   " converts between pointers to integer types with different sign">,
-  InGroup<DiagGroup<"pointer-sign">>;
+  InGroup<DiagGroup<"pointer-sign">>, DefaultIgnore;
 def ext_typecheck_convert_incompatible_pointer : ExtWarn<
   "incompatible pointer types "
   "%select{%diff{assigning to $ from $|assigning to different types}0,1"
